### PR TITLE
feat: prepare 1.5.1.1 release

### PR DESCRIPTION
**Pull request by CumpsD**
Prepare the 1.5.1.1 release to bump npgsql version

- [x] Added release notes
- [x] Tested build script

Did some minor formatting changes while at it, to keep the Build.Props clean since FAKE likes to rewrite the entire file when it touches 1 thing

### CODE REVIEW

File: src/Directory.Generated.props
Comment (Aaronontheweb) [inline src/Directory.Generated.props:1]: What's this file for?

**Review by Aaronontheweb — COMMENTED** [review]
What's the generated.props for?

File: src/Directory.Generated.props
Comment (CumpsD) [reply src/Directory.Generated.props:1]: It's where FAKE writes the version number and release notes in. The "issue" is FAKE likes to overwrite and reformat the entire file when it writes into it. So this file is especially for FAKE, and is referenced from Build.Props (like I did in the Persistence Sql repo)

Comment (Aaronontheweb) [reply src/Directory.Generated.props:1]: interesting

Comment (CumpsD) [reply src/Directory.Generated.props:1]: It's referenced with: 
```<Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Generated.props', '$(MSBuildThisFileDirectory)'))" />```

and FAKE writes to it with:
```
XmlPokeInnerText "./src/Directory.Generated.props" "//Project/PropertyGroup/VersionPrefix" releaseNotes.AssemblyVersion
XmlPokeInnerText "./src/Directory.Generated.props" "//Project/PropertyGroup/PackageReleaseNotes" (releaseNotes.Notes |> String.concat "\n")
```

**Pull request merged**